spec/view/post_index_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe 'Post index', type: :feature do
+  before(:each) do
+    @user = User.create(
+      name: 'Tarek',
+      bio: 'I am a full-stack developer',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      post_counter: 0
+    )
+
+    @post1 = Post.create(
+      author: @user,
+      title: 'My first post',
+      text: 'This is my first post',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+
+    @post2 = Post.create(
+      author: @user,
+      title: 'My second post',
+      text: 'This is my second post',
+      comments_counter: 0,
+      likes_counter: 0
+    )
+
+    Comment.create(post_id: @post1.id, author_id: @user.id, text: 'Great job!')
+    Comment.create(post_id: @post2.id, author_id: @user.id, text: 'Thanks for the effort')
+    Like.create(post_id: @post1.id, author_id: @user.id)
+    Like.create(post_id: @post2.id, author_id: @user.id)
+
+    visit user_posts_path(@user)
+  end
+
```

Comment:
- [ ] kindly fix your test as some are failing
 
![image](https://user-images.githubusercontent.com/66097371/200780686-5b695aea-b90a-4536-820f-ecc11f265417.png)

Reply:
@VICTOR-CHUKWU Thank you for your reply, I am having the tests pass on my pc 
![image](https://user-images.githubusercontent.com/83262890/200783443-31d7ca52-d941-47e2-a235-427c0b71aa17.png)

Can you send me a screenshot of one of the failing tests, to know the error?

Reply:
@tarek102  I hope its clear now after the zoom call, Kindly make the changes and the next reviewer should approve :clap:

Reply:
Thanks for the zoom meeting, really appreciate it.